web-registry/src/components/templates/ProjectDetails/ProjectDetails.tsx

```diff
@@ -144,7 +155,8 @@ function ProjectDetails(): JSX.Element {
   });
   const mediaData = useMedia({ metadata, geojson });
   const impactData = useImpact({ coBenefitsIris, primaryImpactIRI });
-  const isLoading = loading || loadingDataByHandle;
+
+  const isLoading = loadingProjectByOnChainId || loadingProjectByHandle;
```

Comment:
Something about the loading logic is preventing the `<Skeleton>`s  from resolving. 

Note the difference between Dev: https://dev--regen-registry.netlify.app/project/C01-007
and this preview: https://deploy-preview-1689--regen-registry.netlify.app/project/C01-007

In this C01-007 case, we have no iri-resolved-metadata _or_ postgres data, but the on-chain entity exists, so we should be able to see this:

<img width="1616" alt="Screen Shot 2022-12-20 at 2 04 56 PM" src="https://user-images.githubusercontent.com/9438198/208766334-2c5ca89f-6947-4a67-abe5-12551d8d028b.png">

Reply:
Nice catch, the issue here was the skeleton was using the `isLoading` flag from `react-query`. This flag stays false until there's data in the cache which does not work well with disabled queries. For queries that can be disabled, it's best to use the `isInitialLoading` flag, a combination of `isLoading && isFetching`.